qemu/tests/sev_hotplug_mem.py

```diff
@@ -0,0 +1,66 @@
+import logging
+import time
+
+from avocado import TestError
+from virttest import error_context
+from virttest.utils_test.qemu import MemoryHotplugTest
+
+
+LOG_JOB = logging.getLogger('avocado.test')
+
+
+class MemoryHotplugSimple(MemoryHotplugTest):
+
+    def check_memory(self, vm, wait_time=0):
+        """
+        Check is guest memory is really match assigned to VM.
+
+        :param vm: VM object, get VM object from env if vm is None.
+        :param wait_time: time os waits after hotplug/unplug default is 0.
+        """
+        error_context.context("Verify guest memory size", LOG_JOB.info)
+        threshold = float(self.params.get("threshold", 0.10))
+        time.sleep(wait_time)
+        vm_mem_size = super(MemoryHotplugSimple, self).get_guest_total_mem(vm)
+        assigned_vm_mem_size = super(MemoryHotplugSimple, self).get_vm_mem(vm)
```

Comment:
hi Zixi, it's OK to call the functions from parent class with self.function(), super() can be used for calling the  function with the same name, e.g. in your case, if you wanna call parent's check_memory, then super() is needed

Reply:
Thanks Zhenchao for the explanation, I re-wrote check_memory(), can't call parent's function.

Reply:
Thanks, Zhenchao, I get it now, it works for me. 

python3 ConfigTest.py --testcase=sev_hotplug_mem --guestname=RHEL.9.4.0   --platform=x86_64 --clone=no --profile=profile 
(1/4) Host_RHEL.m9.u4.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.4.0.x86_64.io-github-autotest-qemu.sev_hotplug_mem.backend_ram.hotplug.q35: STARTED
 (1/4) Host_RHEL.m9.u4.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.4.0.x86_64.io-github-autotest-qemu.sev_hotplug_mem.backend_ram.hotplug.q35: PASS (115.98 s)
 (2/4) Host_RHEL.m9.u4.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.4.0.x86_64.io-github-autotest-qemu.sev_hotplug_mem.backend_ram.unused_memory.q35: STARTED
 (2/4) Host_RHEL.m9.u4.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.4.0.x86_64.io-github-autotest-qemu.sev_hotplug_mem.backend_ram.unused_memory.q35: PASS (141.25 s)
 (3/4) Host_RHEL.m9.u4.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.4.0.x86_64.io-github-autotest-qemu.sev_hotplug_mem.backend_file.hotplug.q35: STARTED
 (3/4) Host_RHEL.m9.u4.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.4.0.x86_64.io-github-autotest-qemu.sev_hotplug_mem.backend_file.hotplug.q35: PASS (121.05 s)
 (4/4) Host_RHEL.m9.u4.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.4.0.x86_64.io-github-autotest-qemu.sev_hotplug_mem.backend_file.unused_memory.q35: STARTED
 (4/4) Host_RHEL.m9.u4.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.4.0.x86_64.io-github-autotest-qemu.sev_hotplug_mem.backend_file.unused_memory.q35: PASS (145.89 s)